include/librealsense2/h/rs_option.h

```diff
@@ -76,6 +76,9 @@ extern "C" {
         RS2_OPTION_HARDWARE_PRESET, /**< Hardware stream configuration */
         RS2_OPTION_GLOBAL_TIME_ENABLED, /**< disable global time  */
         RS2_OPTION_APD_TEMPERATURE, /**< APD temperature*/
+        RS2_OPTION_PROJECTOR_TYPE, /**< Select between the projector and LED on F416 camera, 0 - for projector, 1 - for LED*/
+        RS2_OPTION_LED_PWR_MODE, /**< Set the LED on or off on F416 camera*/
```

Comment:
Why not RS2_OPTION_LED_~~PWR~~_MODE ? The PRW  can be confusing with the next enum